sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -138,6 +138,11 @@ class ParquetFileFormat
       conf.setBoolean(ParquetOutputFormat.ENABLE_JOB_SUMMARY, false)
     }
 
+    require(!conf.getBoolean(ParquetOutputFormat.ENABLE_JOB_SUMMARY, false)
+      || classOf[ParquetOutputCommitter].isAssignableFrom(committerClass),
+      s"Committer $committerClass is not a ParquetOutputCommitter and cannot create job summaries."
+    + " Set Parquet option " + ParquetOutputFormat.ENABLE_JOB_SUMMARY + " to false.")
```

Comment:
nit:

```scala
  ...
  s"Committer $committerClass is not a ParquetOutputCommitter and cannot create job summaries. " +
  s"Set Parquet option '${ParquetOutputFormat.ENABLE_JOB_SUMMARY}' to false.")
```

Reply:
I'd thought about that; didn't look any better or worse. Will change it for log message.